src/components/file-selector/file-selector.scss

```diff
@@ -36,6 +36,7 @@
 
 .sbb-file-selector__input-container {
   -webkit-tap-highlight-color: transparent;
+  line-height: 0;
```

Comment:
Would you happen to know why it's needed? I did some quick tests and did not find it :/

Reply:
Without it, there's a small delta in height when checking diffs with figma:
<img width="671" alt="image" src="https://github.com/sbb-design-systems/lyne-components/assets/101575400/e0c71729-f37b-48aa-8cf9-0a49997e0fa9">

Reply:
could it maybe be solved by using display flex?

Reply:
Strange :| Nothing changes on my browser if I remove it